applications/CoSimulationApplication/custom_io/co_sim_EMPIRE_API.h

```diff
@@ -107,7 +109,7 @@ static void SendArray(const std::string& rFileName, const int sizeOfArray, const
 
     output_file << std::scientific << std::setprecision(12); // TODO maybe this should be configurable
```

Comment:
just for the record, maybe using `std::defaultfloat` will give a smaller file-size, we can try in the future